sql/mysql/migrate.go

```diff
@@ -47,6 +51,33 @@ func (m *migrate) Exec(ctx context.Context, changes []schema.Change) error {
 	return nil
 }
 
+// topLevel executes first the changes for creating or dropping schemas (top-level schema elements).
+func (m *migrate) topLevel(ctx context.Context, changes []schema.Change) ([]schema.Change, error) {
```

Comment:
I thought about moving it out to `sqlx`, but meh. too many abstractions for saving 10 lines of code.

![image](https://user-images.githubusercontent.com/7413593/145708712-fea7a331-12e7-441b-87ac-6e9ba4caa1de.png)

![image](https://user-images.githubusercontent.com/7413593/145708717-7bbc65b0-038d-43a0-91a4-0eb8b4463a2d.png)